srtcore/api.h

```diff
@@ -123,6 +125,8 @@ class CUDTSocket
 
    srt::sync::Mutex m_ControlLock;           //< lock this socket exclusively for control APIs: bind/listen/connect
 
+   CUDT& core() { return *m_pUDT; }
+
```

Comment:
"core" is rather a bad name. But not sure which name is better :(

Reply:
Still, exposing a method instead of reaching out to the internal field looks at least better.